File: src/test/java/com/andy/StudentTest.java
package com.andy;

import com.andy.student.Student;
import org.junit.jupiter.api.Assertions;
import org.junit.jupiter.api.Test;

public class StudentTest {
    @Test
    public void highestScoreTest(){
        Student student = new Student("andy",60,80);
        Assertions.assertEquals(80,student.highest());
    }

    @Test
    public void averageTest(){
        Student student = new Student("andy",60,80);
        Assertions.assertEquals(60+80/2, student.getAverage());
    }

}

File: src/test/kotlin/com/andy/StudentTest.kt
package com.andy

import com.andy.studenKt.Student
import org.junit.jupiter.api.Assertions
import org.junit.jupiter.api.Test

class StudentTest{

    @Test
    fun highestTest(){
        val student = Student("andy",60 , 80)
        Assertions.assertEquals(80,student.highest())
    }

    @Test
    fun averageTest(){
        val student = Student("andy",60 , 80)
        Assertions.assertEquals((60+80)/2, student.getAverage())
    }

    @Test
    fun gradingTest(){
        val student = Student("andy",60 , 80)
        Assertions.assertEquals("C", student.grading())
    }
}